Server/src/main/java/org/openas2/cmd/processor/restapi/ApiResource.java

```diff
@@ -5,30 +5,22 @@
  */
 package org.openas2.cmd.processor.restapi;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import jakarta.ws.rs.*;
```

Comment:
Please do not use asterisk imports, import every class spciofically.
If you want to know why I prefer this way, do an internet search "using wildcard import in java".